Lanchonete/LanchoneteTests/Managers/ShoppingCartManagerTests.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  ShoppingCartManagerTests.swift
+//  LanchoneteTests
+//
+//  Created by vera.dias on 1/22/21.
+//
+
+import XCTest
+@testable import Lanchonete
+
+class ShoppingCartManagerTests: XCTestCase {
+    
+    func test_GIVEN_an_empty_shopping_cart_WHEN_I_add_a_sandwich_THEN_the_total_should_be_the_price_of_the_added_sandwich() {
+        let manager = ShoppingCartManager()
+        //let sandwich = Sandwich(type: .xBacon)
+        manager.addSandwich(type: .xBacon)
```

Comment:
No it's not for testing purposes, this model implementation is very strict and it was created to only have the specific ingredients and the menu sandwiches. This implementation will change in the near future since the sandwiches can be customised and the ingredients can change as well. Hope I helped. 😃

Reply:
But bacon is not model, right? Bacon is a concrete data.
But I got that it's not final, thanks